lms/static/sass/views/_teams.scss

```diff
@@ -472,4 +472,195 @@
       padding-left: 2%;
     }
   }
+
+  .add-team {
+    @extend %ui-well;
+    margin: 20px 1.2%;
+    color: $gray;
+    text-align: center;
+
+    .title {
+      @extend %t-title6;
+      @extend %t-strong;
+      margin-bottom: ($baseline/2);
+      text-align: inherit;
+      color: inherit;
+    }
+
+    .copy {
+      text-align: inherit;
+      color: inherit;
+    }
+  }
 }
+
+.teams-content {
+
+  .create-new-team {
+    padding: $baseline;
+    overflow: hidden;
+
+    .team-required-fields {
+      @include float(left);
+      width: 55%;
+      border-right: 2px solid $gray-l4;;
+
+      .u-field.u-field-name {
+        padding-bottom: $baseline;
+
+        .u-field-value {
+          display: block;
+          width: 90%;
+
+          input {
+            border-radius: ($baseline/5);
+            height: ($baseline*2);
+          }
+        }
+
+        .u-field-message {
+          @include padding-left(0);
+          padding-top: ($baseline/4);
+          width: 100%;
+        }
+      }
+
+      .u-field.u-field-description {
+
+        .u-field-value {
+          display: block;
+          width: 100%;
+
+          textarea {
+            height: ($baseline*5);
+            width: 90%;
+            border-radius: ($baseline/5)
+          }
+        }
+
+        .u-field-description-message {
+          @extend %t-copy-sub1;
+          margin-top: ($baseline/4);
+          color: $gray-l1;
+          width: 90%;
+        }
+      }
+
+      .u-field-title {
+        padding-bottom: ($baseline/4);
+        color: $base-font-color;
+        width: 40%;
+      }
+    }
+
+    .team-optional-fields {
+      @include float(left);
+      @include margin-left($baseline);
+      width: 40%;
+
+      .u-field.u-field-optional_description {
+        margin-bottom: ($baseline/5);
+
+        .u-field-title {
+          color: $base-font-color;
+          font-weight: $font-semibold;
+          margin-bottom: ($baseline/5);
+          width: 100%;
+        }
+
+        .u-field-value {
+          display: none;
+        }
+      }
+
+      .u-field.u-field-language {
+        margin-bottom: ($baseline/5);
+      }
+
+      .u-field-value-display {
+        display: none;
+      }
+
+      .u-field-value {
+        width: 90%;
+      }
+
+      .u-field-title {
+        display: block;
+        color: $base-font-color;
+        width: 35%;
+      }
+
+      .u-field-message {
+        @include padding-left(0);
+        width: 95%;
+      }
+    }
+
+    .u-field {
+      padding: 0;
+    }
+
+    .u-field.error {
+      input, textarea {
+        border-color: $red;
+      }
+
+      .u-field-message-help, .u-field-description-message {
+        color: $red !important;
+      }
+    }
+
+    .wrapper-msg {
+      display: none;
+      margin: 0 0 $baseline 0;
+    }
+  }
+}
+
+@mixin button-base-sytle($text-color) {
```

Comment:
This is not a great place to put a global mixin. Can you add it to the mixins file or put it inside the teams context? And where do you use this? I don't see it called in the code anywhere. If it isn't used, you can just remove it.

Also, there are a few things in here that need to be fixed: 
* a typo in the mixin name "sytle" instead of "style"
* property order
* if this is meant to override/reset the poorly scoped blue button style, can you add a note about that?